scripts/globals/abyssea/lights.lua

```diff
@@ -0,0 +1,652 @@
+        ["Iratham"]                = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Lachrymater"]            = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Lacovie"]                = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Manananggal"]            = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Mictlantecuhtli"]        = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Minhocao"]               = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Muscaliet"]              = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Myrmecoleon"]            = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Ophanim"]                = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Quetzalli"]              = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Rubicund_Adenium"]       = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Tefenet"]                = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Treble_Noctules"]        = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Vetehinen"]              = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+
+        -- VWNM
+        ["Hungerer"]               = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Yearner"]                = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+        ["Usurper"]                = { azure = 16,  pearl = 16,  ruby = 16,  amber = 16 },
+    },
+
+    [xi.zone.ABYSSEA_MISAREAUX] =
+    {
+        ["Abyssobugard"]           = { azure = 8,   pearl = 5,   ruby = 8,   amber = 16 }, -- Bugards
+        ["Atrociraptor"]           = { azure = 8,   pearl = 5,   ruby = 8,   amber = 8  }, -- Raptors
+        ["Ancient_Orobon"]         = { azure = 0,   pearl = 5,   ruby = 16,  amber = 8  }, -- Orobon
+        ["Squib"]                  = { azure = 0,   pearl = 5,   ruby = 8,   amber = 8  }, -- Poroggos
+        ["Coastal_Colibri"]        = { azure = 8,   pearl = 5,   ruby = 0,   amber = 8  }, -- Colibri
+        ["Frigatebird"]            = { azure = 8,   pearl = 5,   ruby = 8,   amber = 8  }, -- Birds
+        ["Slasher"]                = { azure = 8,   pearl = 5,   ruby = 8,   amber = 8  }, -- Crabs
+        ["Overking_Apkallu"]       = { azure = 0,   pearl = 5,   ruby = 0,   amber = 16 }, -- Apkallu
+        ["Buzzfly"]                = { azure = 8,   pearl = 5,   ruby = 8,   amber = 8  }, -- Flies
+        ["Gasher"]                 = { azure = 8,   pearl = 5,   ruby = 8,   amber = 8  }, -- Crabs
+        ["Maritime_Peiste"]        = { azure = 16,  pearl = 5,   ruby = 8,   amber = 8  }, -- Peistes
+        ["Escarp_Murex"]           = { azure = 8,   pearl = 5,   ruby = 0,   amber = 8  }, -- Murex
+        ["Ephemeral_Murex"]        = { azure = 100, pearl = 5,   ruby = 16,  amber = 16 }, -- Murex
```

Comment:
Should be 128 azure
https://ffxiclopedia.fandom.com/wiki/Ephemeral_Murex
https://www.bg-wiki.com/ffxi/Abyssea_Lights

Reply:
Wrong. Retail does not consistently give you 128, nor is this 100 the amount you will actually get. The script uses 50 and 100 as "special" numbers to note a variance/range of results.

_I'll take #shitWikiSays for 100, Ghost of Alex Trebek_

Reply:
Additionally I should mention this is a systems work, clay is taking exactly what we had but making the code easier to deal with future forward - any preexisting error is likely to remain till after he's finished and be worked on separately

Reply:
Yeah number was UP to 128 iirc but I had no proof to back that up.